src/components/App/index.js

```diff
@@ -17,27 +17,18 @@ if (typeof window !== 'undefined') {
   });
 }
 
-export default class App extends Component {
-  state = {
-    isHome: true,
-  };
-
-  handleRoute = (event) => {
-    this.setState({
-      isHome: event.url === routeMap.home,
-    });
-  };
-
-  render() {
-    const { isHome } = this.state;
-    return (
-      <Wrapper isHome={isHome} id='app'>
-        {!isHome && (<Header Dark />)}
-        <Router onChange={this.handleRoute}>
+export default () => (
```

Comment:
It's better to name the component first by assigning it to a value.
Otherwise it is hard to debug it inside **devtools**